contents/docs/plugins/build/tutorial.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Tutorial
+sidebar: Docs
+showTitle: true
+---
+
+This tutorial explains the development workflow and best practices, using a hello world plugin. We go from zero to publishing your plugin in the official PostHog repository.
```

Comment:
nit:
"a hello world plugin" implies "hello world" is some well-known plugin type.

How about something like:

"using an example 'Hello World' plugin"